Cargo.toml

```diff
@@ -13,6 +13,7 @@ description = "Fuel cryptographic primitives."
 [dependencies]
 borrown = "0.1"
 fuel-types = { version = "0.3", default-features = false }
+hex = { version = "0.4.3", default-features = false, features = ["std"] }
```

Comment:
We should import `std` only if `std` is enabled for this crate

Reply:
Not needed since `Bytes64` provides `FromStr`: https://github.com/FuelLabs/fuel-crypto/pull/16#pullrequestreview-919383836